### ISSUE TITLE
edupub implicit body test still needs aria-label

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epubcheck#1109]
There are two valid tests for the body element that differ only in that one has a heading and the other does not.

One is called an "explicit body" and the other an "implict body", but it doesn't make sense that both can be valid as the "implicit" one would still need an aria-label if it doesn't have an explicit heading.

I guess it validates because both tests only use sectioning content as children, so the "implicit body" test doesn't get flagged.

I think we can just drop this tests as there's already an invalid check for an implicit body without a heading and a valid test for a body that only contains sectioning elements.
